.github/workflows/windows-unittests.yml

```diff
@@ -57,5 +57,6 @@ jobs:
           # FIXME: skipping rtloader tests because they fail with a DLL-not-found error
           # inv -e rtloader.test
           inv -e test --skip-linters --rerun-fails=2 --python-runtimes 3 --coverage --profile --python-home-3=$pythonLocation
-          # The Codecov Python uploader was deprecated on Feb 1, 2022 and disappeared from PyPi on Apr 12, 2023.
-          # inv -e codecov
+
+      - name: Upload Codecov results
+        uses: codecov/codecov-action@v3
```

Comment:
I assume all of the defaults work with out we're outputting and saving the coverage file?

Reply:
right now it's not working, that's why I try to change stuff to make it pick the generated coverage report files

Reply:
Ah, apologies - I had thought this was ready for review.